src/CovidAppointmentTable.js

```diff
@@ -60,8 +73,8 @@ export default function CovidAppointmentTable() {
     const [ready, setReady] = useState(false);
     const [errorMessage, setErrorMessage] = useState();
     const [sortInfo, setSortInfo] = useState({
-        sortKey: "hasAppointments",
-        sortAsc: false,
+        sortKey: "location",
```

Comment:
Was sorting by location an intended change here?

Reply:
Yes, sorry. I forgot I snuck it in. I put it at the end of the PR description (first comment). LMK if you'd prefer I remove it and add it separately

Reply:
all good! no need to create a separate PR